packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -240,9 +240,14 @@ export class KubernetesClient {
         }
       }
     } catch (err) {
-      const namespaces = await ctx.makeApiClient(CoreV1Api).listNamespace();
-      if (namespaces?.body?.items.length > 0) {
-        namespace = namespaces?.body?.items[0].metadata?.name;
+      try {
+        const namespaces = await ctx.makeApiClient(CoreV1Api).listNamespace();
+        if (namespaces?.body?.items.length > 0) {
+          namespace = namespaces?.body?.items[0].metadata?.name;
+        }
+      } catch (error) {
+        // unable to list namespaces, can be due to a connection refused (cluster not up)
+        console.trace('unable to list namespaces', error);
```

Comment:
So it will trace the error, but no erroring out?

I'm assuming it's fine for it to just revert back to `default`. but this is outside of the scope of the PR.